dev-java/commons-vfs/files/commons-vfs-2.0-utf8.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/main/java/org/apache/commons/vfs2/FileSystemManager.java b/src/main/java/org/apache/commons/vfs2/FileSystemManager.java
+index bb8cc2a..1e26559 100644
+--- a/src/main/java/org/apache/commons/vfs2/FileSystemManager.java
++++ b/src/main/java/org/apache/commons/vfs2/FileSystemManager.java
+@@ -98,7 +98,7 @@ public interface FileSystemManager
+     FileObject resolveFile(String name, FileSystemOptions fileSystemOptions)
+         throws FileSystemException;
+ 
+-    /**                               �
++    /**
```

Comment:
what is this patch about?

Reply:
> what is this patch about?

That character is invalid for utf8 and would force us setting java encoding to `iso-8859-1`